docs/swagger.yml

```diff
@@ -1484,19 +1511,23 @@ definitions:
           systems_count:
             type: "integer"
             format: "int64"
-            description: "The total number of systems to be scanned."
+            description: "The total number of systems to be scanned"
+            example: 20
           systems_scanned:
             type: "integer"
             format: "int64"
-            description: "The total number of systems that have been scanned."
+            description: "The total number of systems that have been scanned"
+            example: 10
           systems_failed:
             type: "integer"
             format: "int64"
-            description: "The total number of systems that failed to scan."
+            description: "The total number of systems that failed to scan"
+            example: 10
           report_id:
             type: "integer"
             format: "int64"
-            description: "The identifier of the sent fact collection."
+            description: "The identifier of the sent fact collection"
```

Comment:
Just a suggestion but maybe change this to `The fact collection identifier` or `Fact collection identifier` to be consistent with line 2017.